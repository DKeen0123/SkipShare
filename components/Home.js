import React, { Component } from 'react'
import { Image, View, Dimensions, Button } from 'react-native'
import navStyles from '../styles/navStyles';
import LogoTitle from './LogoTitle';

export default class Home extends Component {
  static navigationOptions = {
    headerTitle: <LogoTitle/>,
    ...navStyles
  }

  viewListings = () => {

  }

  createListing = () => {
    this.props.navigation.navigate('CreateListing');
  }

  render() {
    return (
      <View>
        <Image
          source={require('../images/skipshare.png')}
          style={{width: Dimensions.get('window').width}}
        />
        <Button
          title='View listings'
          onPress={this.viewListings}
        />
        <Button
          title='Create a new listing'
          onPress={this.createListing}
        />
      </View>
    )
  }
}