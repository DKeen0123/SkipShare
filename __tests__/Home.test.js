import React from 'react';
import { shallow } from 'enzyme';
import Home from '../components/Home';

const home = shallow(<Home />);

it('renders without crashing', () => {
  expect(home).toMatchSnapshot();
});