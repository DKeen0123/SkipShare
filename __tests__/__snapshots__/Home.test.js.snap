// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Component>
  <Image
    source={1}
    style={
      Object {
        "width": 750,
      }
    }
  />
  <Button
    onPress={[Function]}
    title="View listings"
  />
  <Button
    onPress={[Function]}
    title="Create a new listing"
  />
</Component>
`;
